core/background/main.js

```diff
@@ -80,7 +80,7 @@ require([
 
 			if (!isActiveSession) {
 				isActiveSession = true;
-				GA.send('pageview', '/background-injected?version=' + chrome.app.getDetails().version);
+				GA.send('pageview', '/background-injected?version=' + chrome.runtime.getManifest().version);
```

Comment:
Don't know about the example, but `chrome.runtime.getManifest` works fine as well as other Chrome API functuions (those, that implemented in Firefox, of course).